docs/root/intro/version_history.rst

```diff
@@ -21,7 +22,7 @@ Version history
 * stream: renamed the `RequestInfo` namespace to `StreamInfo` to better match
   its behaviour within TCP and HTTP implementations.
 * thrift_proxy: introduced thrift rate limiter filter
-* upstream: changed how load calculation for :ref:`priority levels<arch_overview_load_balancing_priority_levels>` and :ref:`panic thresholds<arch_overview_load_balancing_panic_threshold>` interact. As long as normalized total health is 100% panic thresholds are disregarded. 
+* upstream: changed how load calculation for :ref:`priority levels<arch_overview_load_balancing_priority_levels>` and :ref:`panic thresholds<arch_overview_load_balancing_panic_threshold>` interact. As long as normalized total health is 100% panic thresholds are disregarded.
```

Comment:
My editor (vscode) is set to to trim trailing spaces on save, which is why these changes slipped in. Are you fine with them, does it affect the `.rst` parsing somehow? If it does, let me know, otherwise I think it's better if we can keep the entries without trailing spaces if possible.